packages/image_picker/example/test_driver/test/image_picker_e2e_test.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019, the Chromium project authors.  Please see the AUTHORS file
```

Comment:
This is the test harness, but I'm not actually seeing specific e2e test cases in this PR. Would it make sense to try and test this more?

Reply:
All the image picker methods are static and the Future won;'t return until some sort of user interaction. So it might be hard to test at the current state.

Reply:
Ah, right. In that case having it at least test the binding is better than nothing and makes sense, thanks.